aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/deploying-web-packages.md

```diff
@@ -114,11 +114,54 @@ MSDeploy.exe relies on [Web Deploy providers](https://technet.microsoft.com/libr
 
 In addition, you'll need to specify various other [provider-specific settings](https://technet.microsoft.com/library/dd569001(WS.10).aspx) and general [operation settings](https://technet.microsoft.com/library/dd569089(WS.10).aspx). For example, suppose you want to deploy the ContactManager.Mvc web application to a staging environment. The deployment will target the Web Deploy Handler and must use basic authentication. To deploy the web application, you need to complete the next steps.
 
-**To deploy a web application using MSDeploy.exe**
+**To deploy a web application using MSDeploy.exe using an access token**
+
+MSDeploy V3 supports authentication with an access token, also known as a bearer token. If available, it is recommended to use an access token for best security.
```

Comment:
```suggestion
MSDeploy V3 supports authentication with an access token, also known as a bearer token. Access tokens are recommended because they are the most secure.
```